android/src/main/java/com/reactnativecommunity/picker/ReactPickerManager.java

```diff
@@ -180,6 +186,18 @@ public void receiveCommand(@NonNull ReactPicker root, String commandId, @android
     }
   }
 
+  public void focus(ReactPicker root) {
+    root.performClick();
+  }
+
+  public void blur(ReactPicker root) {
+    root.clearFocus();
+  }
+
+  public void setNativeSelected(ReactPicker picker, int selected) {
+    picker.setStagedSelection(selected);
+  }
```

Comment:
We could use those methods in the `receiveCommands` and `setSelected` prop to avoid duplication.